### PR TITLE
Update Team Page

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

**Comment by CalvinKirs** [comment]
image

**Comment by CalvinKirs** [comment]
image

File: src/pages/team/languages.json [hunk 1]
```diff
@@ -1,205 +1,115 @@
...
       },
       {
         "apacheId": "",
-        "avatarUrl": "https://avatars.githubusercontent.com/u/15833811?v=4",
-        "email": "lidongdai@apache.org",
-        "gitUrl": "https://github.com/dailidong",
+        "avatarUrl": "https://avatars.githubusercontent.com/u/158903?v=4",
+        "email": "jbonofre@apache.org",
+        "gitUrl": "https://github.com/jbonofre",
         "githubId": "dailidong",
```
Comment (wolfboys) [inline src/pages/team/languages.json:45]: githubid: jbonofre

File: src/pages/team/languages.json [hunk 2]
```diff
@@ -208,202 +118,112 @@
...
```
Comment (wolfboys) [inline src/pages/team/languages.json:159]: githubid: jbonofre

**Review by wolfboys — COMMENTED** [review]
some githubId is wrong

**Comment by CalvinKirs** [comment]
> some githubId is wrong

thx, done

**Review by zhongjiajie — APPROVED** [review]
lgtm

**Pull request merged**